packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -255,6 +249,24 @@ const FormAds = observer(() => {
                     );
                 }}
             </Formik>
+            <Modal
+                className='p2p-my-ads__modal-error'
+                is_open={!!my_ads_store?.api_error_message}
```

Comment:
Both the casting (`!!`) and optional chaining (`?`) are not required. Any `api_error_message` will be truthy and `api_error_message` always exists on the `my_ads_store` instance.